origin.spec

```diff
@@ -49,6 +49,315 @@ Requires:       %{name}-clients = %{version}-%{release}
+Provides: golang(bundled(k8s.io/kubernetes/plugin/pkg/admission/initialresources)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/plugin/pkg/admission/limitranger)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/plugin/pkg/admission/namespace/autoprovision)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/plugin/pkg/admission/namespace/exists)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/plugin/pkg/admission/namespace/lifecycle)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/plugin/pkg/admission/resourcequota)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/plugin/pkg/admission/securitycontext/scdeny)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/plugin/pkg/admission/serviceaccount)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/plugin/pkg/auth/authenticator/password/passwordfile)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/plugin/pkg/auth/authenticator/request/basicauth)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/plugin/pkg/auth/authenticator/request/keystone)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/plugin/pkg/auth/authenticator/request/union)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/plugin/pkg/auth/authenticator/request/x509)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/plugin/pkg/auth/authenticator/token/oidc)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/plugin/pkg/auth/authenticator/token/tokenfile)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/plugin/pkg/scheduler)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/test/e2e)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/third_party/forked/coreos/go-etcd/etcd)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/third_party/forked/json)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/third_party/forked/reflect)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/third_party/golang/expansion)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/third_party/golang/netutil)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(k8s.io/kubernetes/third_party/golang/template)) = 4c8e6f47ec23f390978e651232b375f5f9cde3c7
+Provides: golang(bundled(speter.net/go/exp/math/dec/inf)) = 42ca6cd68aa922bc3f32f1e056e61b65945d9ad7
+
```

Comment:
This will have to be maintained outside of the spec file as we're not (currently) updating the spec file on each release. Perhaps we should have tito do this too?